test/react/fetcher-type.test.tsx

```diff
@@ -1,10 +1,10 @@
-import { unstable_createRemixStub as createRemixStub } from "@remix-run/testing";
 import { render, screen } from "@testing-library/react";
 // @vitest-environment happy-dom
 import * as React from "react";
+import { createRoutesStub } from "react-router";
 import { describe, expect, test } from "vitest";
 
-import { useFetcher } from "@remix-run/react";
+import { useFetcher } from "react-router";
```

Comment:
This import can be merged with the react-router import above

Reply:
yeah I was using patterns to rename so there might be a few of these around, I think biome can also handle this on it's own if configured, but I might be wrong!